modules/signals/src/signal-store.ts

```diff
@@ -325,16 +325,14 @@ export function signalStore(
         (this as any)[key] = props[key];
       }
 
-      if (hooks.onInit) {
-        hooks.onInit();
-      }
+      const { onInit, onDestroy } = hooks;
 
-      if (hooks.onDestroy) {
-        const injector = inject(Injector);
+      if (onInit) {
+        onInit();
+      }
 
-        inject(DestroyRef).onDestroy(() => {
-          runInInjectionContext(injector, hooks.onDestroy!);
```

Comment:
`Injector` and `runInInjectionContext` APIs are unused and can be removed from imports

Reply:
Done